doc/services/file_system/index.rst

```diff
@@ -51,10 +51,10 @@ Samples for the VFS are mainly supplied in ``samples/subsys/fs``, although vario
 VFS usage are provided as important functionalities in samples for different subsystems.
 Here is the list of samples worth looking at:
 
-- ``samples/subsys/fs/fat_fs`` is an example of FAT file system usage with SDHC media;
-- ``samples/subsys/shell/fs`` is an example of Shell fs subsystem, using internal flash partition
+- :zephyr:code-sample:`fs` is an example of FAT file system usage with SDHC media;
+- :zephyr:code-sample:`shell-fs` is an example of Shell fs subsystem, using internal flash partition
 	formatted to LittleFS;
-- ``samples/subsys/usb/mass/`` example of USB Mass Storage device that uses FAT FS driver with RAM
+- :zephyr:code-sample:`usb-mass` example of USB Mass Storage device that uses FAT FS driver with RAM
 	or SPI connected FLASH, or LittleFS in flash, depending on the sample configuration.
```

Comment:
Thanks for the quick turnaround
as you may have noticed, current list is actually kind of messed up styling wise https://docs.zephyrproject.org/latest/services/file_system/index.html#samples
You might as well fix it if you don't mind -- please replace the tabs at line 56 and 58 by two spaces so that the continuation of each paragraph is correct

Thanks again!

Reply:
Yes, found the two items text was bold. Done.

Reply:
Thanks, perfect!
Just for the record they were rendering as such due to being flagged as "definition list" https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists hence why the first line was bold as the parser was taking it as the "definition" :)